docs/api.rst

```diff
@@ -158,7 +158,7 @@ In order to check that a build id exists, we'll just check that it is mentioned
 
 .. code-block:: bash
 
-    curl -s $SERVER/buckets/build-hub/collections/releases/search?q=build.id=20170713200529 | \
+    curl -s $SERVER/buckets/build-hub/collections/releases/search?q=build.id="20170713200529" | \
```

Comment:
Actually I don't think it is the case for the search endpoint is it?

Reply:
It does, and it's less confusing I think